src/pages/api/author.ts

```diff
@@ -1,11 +1,25 @@
 import authorConfig from "../../config/info";
+import { FeatureFlagsApi } from "../../config/featureFlag";
 
 export async function GET() {
   try {
     const responseData = {
       author: authorConfig,
     };
 
+    if (!FeatureFlagsApi.enableauthorInfoAPI) {
```

Comment:
if newsletter is disable the I want to disable that API as well and if Blogs are disable then disable that API as well. but Notion API is under development will be separate.  Once it will be ready then I will merge this into contact form Page Feature Flag but for now. Disable it.